Docs/reference/content/reference/driver/authentication.md

```diff
@@ -30,10 +30,25 @@ Or via the connection string:
 mongodb://username:password@myserver/databaseName
 ```
 
-This is the recommended approach as it will make upgrading from MongoDB 2.6 to MongoDB 3.0 seamless, even after [upgrading the authentication schema](http://docs.mongodb.org/manual/release-notes/3.0-scram/#upgrade-mongodb-cr-to-scram).
+These are the recommended approaches as it will make upgrading from MongoDB 2.6 to MongoDB 3.0 seamless, before and after [upgrading the authentication schema](http://docs.mongodb.org/manual/release-notes/3.0-scram/#upgrade-mongodb-cr-to-scram). For MongoDB 4.0, the above approaches will automatically determine which version of SCRAM should be used (SCRAM-SHA-1 or SCRAM-SHA-256).
 
 {{% note %}}The databaseName part of the connection string indicates which database the credentials are located in. See the [connection string section]({{< relref "connecting.md#connection-string" >}}) for more information on connection strings.{{% /note %}}
 
+### SCRAM
+
+When connecting to a MongoDB 4.0 server without specifying an authentication mechanism, the driver will negotiate with the server to determine whether SCRAM-SHA-1 or SCRAM-SHA-256 is the appropriate mechanism.
+
+#### SCRAM-SHA-256
+
+SCRAM-SHA-256 is the default authentication mechanism chosen as long as the user's authentication mechanism supports it. (See the [mechanism parameter of createUser()](https://docs.mongodb.com/manual/reference/method/db.createUser/#db.createUser) and the ["Supported Authentication Methods: Defaults" section of the Driver Authentication Specification](https://github.com/mongodb/specifications/blob/master/source/auth/auth.rst#defaults) for additional information.
+
+##### SecureStrings
+
+SecureStrings are less secure when used in conjunction with SCRAM-SHA-256, due to the need to temporarily store the cleartext password in a managed memory in order to SASLPrep it. This behavior is no different from other drivers in languages with managed memory. (SCRAM-SHA-1 is *not* affected.) 
```

Comment:
s/less secure/slightly less secure/

s/password in a managed memory/password in a managed memory string/